src/main/webapp/js/selectors/index.js

```diff
@@ -35,6 +35,12 @@ const mapToList = (
 
 const getRelatedField = (state, { relatedField }) => relatedField;
 
+const getIdList = (state, { value }) => _.map(value, item => item.id);
```

Comment:
this will be list of string (ids) not the objects, so you can just return the value here without mapping